docs/data/tree-view/tree-item-customization/tree-item-customization.md

```diff
@@ -0,0 +1,70 @@
+---
+productId: x-tree-view
+title: Tree Item Customization
+components: SimpleTreeView, RichTreeView, TreeItem, TreeItem2
+packageName: '@mui/x-tree-view'
+githubLabel: 'component: tree view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+---
+
+# Tree Item Customization
+
+<p class="description">Learn how to customize the Tree Item component.</p>
+
+## Basics
+
+### Change nested item's indentation
+
+Use the `itemChildrenIndentationPx` prop to change the indentation of the nested items.
+By default, a nested item is indented by `12px` from its parent item.
+
+{{"demo": "ItemChildrenIndentationPxProp.js"}}
+
+:::success
```

Comment:
I was super explicit with those `:::success`
This one will be simplified in v8 once we drop the experimental feature and the other one will just disappear with all the doc around the experimental feature.